examples/petsc/bpsraw.c

```diff
@@ -232,7 +232,7 @@ bpData bpOptions[6] = {
     .error = Error3,
     .setupgeofname = SetupDiffGeo_loc,
     .setuprhsfname = SetupDiffRhs3_loc,
-    .applyfname = Diff_loc,
+    .applyfname = Diff3_loc,
```

Comment:
Oh wow, thanks for spotting this!

Reply:
It might be the cause of #451. Let me check.

Reply:
Nope

Reply:
:/